dist.ts

```diff
@@ -0,0 +1,159 @@
+import { Temporal } from "@js-temporal/polyfill";
+import {
+  BASELINE_LOW_TO_HIGH_DURATION,
+  computeBaseline,
+} from "compute-baseline";
+import { Compat, Feature } from "compute-baseline/browser-compat-data";
+import assert from "node:assert/strict";
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import YAML from "yaml";
+
+/**
+ * Update (or create) a dist YAML file from a feature definition YAML file.
+ *
+ * @param {string} fp The path to the human-authored YAML file.
+ */
+function updateDistFile(fp: string): void {
+  const { dir, name: id } = path.parse(fp);
+  const distPath = path.join(dir, `${id}.dist.yml`);
+
+  const stringify = (y: YAML.Document) => y.toString({ lineWidth: 0 });
+
+  const currentDist = fs.existsSync(distPath)
+    ? YAML.parseDocument(fs.readFileSync(distPath, { encoding: "utf-8" }))
+    : undefined;
+  const newDist = toDist(fp);
+
+  currentDist.commentBefore = undefined; // Suppress the generated comment for comparisons
+
+  // Write a dist only if it's new or modified
+  if (!currentDist || stringify(currentDist) !== stringify(newDist)) {
+    newDist.commentBefore = [
+      `Generated from: ${id}.yml`,
+      `Do not edit this file by hand. Edit the source file instead!`,
+      `Last modified: ${Temporal.Now.zonedDateTimeISO("UTC")}`,
```

Comment:
This date is going to lead to merge conflicts any time there are two reviews in flight. Let's use `git log` to find the last modified date instead, when we need it.

Reply:
Fixed, see: https://github.com/web-platform-dx/web-features/pull/787/files#r1553521058.